### ISSUE TITLE
Please include F-Droid as one of the options for getting Mozilla Klar.

### ISSUE THREAD

**Issue by eighthave** [mozilla-mobile/focus-android#1405]
Firefox Klar has been included in F-Droid since 2017-07-04, built directly from source using the official Mozilla build system. As with all apps included in f-droid.org, we also run our automatic verification process on it, you can see Klar diffs here:

    https://verification.f-droid.org/org.mozilla.klar_11.apk.diffoscope.html
    https://verification.f-droid.org/org.mozilla.klar_10.apk.diffoscope.html

Once the last reproducibility issues are fixed, like #1401, then the APK that f-droid.org ships will be exactly the same, except for the APK signature.

It is also possible to include the Mozilla APK signature in f-droid.org provided that Klar builds reproducibly. We test this by building from source on our own infrastructure, then copying the developer's APK signature into our APK. If it verifies, we ship it.

**Comment by bbinto** [comment]
Do we know how many users we will be supporting by doing this?

**Comment by eighthave** [comment]
A couple of years ago, we turned off the weblogs.  We had about 50
millions app downloads from the main f-droid.org app repository at that
time.  Its hard to say more from the f-droid side of things since we
actively avoid tracking users.  Now that we've implemented a bunch of
tracking protections, we plan on adding back some truly anonymous usage
tracking:
https://guardianproject.info/2017/06/08/tracking-usage-without-tracking-people/

**Comment by bbinto** [comment]
To confirm @eighthave - you'd like us to promote it here? https://www.mozilla.org/en-US/firefox/focus/

**Comment by pocmo** [comment]
It's about adding a badge (next to the Google Play one) to the README that is shown here:
https://github.com/mozilla-mobile/focus-android

**Comment by bbinto** [comment]
@eleemoz -- Elvin, is that ok?

**Comment by eighthave** [comment]
Yes, that would be amazing!  I initially asked about getting it on the
Github README, but yes, ideally the F-Droid badge would be next to the
Google Play badge wherever those are used.

**Comment by easyKL** [comment]
F-Droid badge still not there 😞

**Comment by eighthave** [comment]
we would love to have a build flavor that builds Focus without proprietary libs, 
so we can ship it on F-Droid.

**Comment by cadeyrn** [comment]
The original issue was only about adding a badge to the README file of this repository. If you say that you need a different build flavour to ship Firefox Klar on F-Droid then the assumption from the issue description ("the APK that f-droid.org ships will be exactly the same, except for the APK signature") is no longer true. It's probably better to open a separate issue for that.

**Comment by J0WI** [comment]
There depends on #4182

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1802034

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
